src/libOpenImageIO/imagebufalgo_draw.cpp

```diff
@@ -644,14 +644,18 @@ resolve_font (int fontsize, string_view font_, std::string &result)
         if (systemRoot.size())
             font_search_dirs.push_back (std::string(systemRoot) + "/Fonts");
         font_search_dirs.emplace_back ("/usr/share/fonts");
+        font_search_dirs.emplace_back ("/usr/share/fonts/OpenImageIO");
         font_search_dirs.emplace_back ("/Library/Fonts");
         font_search_dirs.emplace_back ("C:/Windows/Fonts");
         font_search_dirs.emplace_back ("/usr/local/share/fonts");
+        font_search_dirs.emplace_back ("/usr/local/share/fonts/OpenImageIO");
         font_search_dirs.emplace_back ("/opt/local/share/fonts");
+        font_search_dirs.emplace_back ("/opt/local/share/fonts/OpenImageIO");
         // Try $OPENIMAGEIOHOME/fonts
         string_view oiiohomedir = Sysutil::getenv ("OPENIMAGEIOHOME");
         if (oiiohomedir.size())
             font_search_dirs.push_back (std::string(oiiohomedir) + "/fonts");
+            font_search_dirs.push_back (std::string(oiiohomedir) + "/share/fonts/OpenImageIO");
```

Comment:
I think we both missed one more... a few lines below this, 

    font_search_dirs.push_back (path+"/fonts");

should be

    font_search_dirs.push_back (path+"/shared/fonts/OpenImageIO");

right?